{"metadata":{"roleHeading":"Enumeration Case","title":"SMIChallengeReasonMalformed","fragments":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"SMIChallengeReasonMalformed"}],"modules":[{"name":"SMIClientCore"}],"symbolKind":"case","externalID":"c:@E@SMIChallengeReason@SMIChallengeReasonMalformed","role":"symbol"},"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonmalformed"]}],"abstract":[{"type":"text","text":"The passed in token was malformed."}],"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}],"languages":["occ"]}],"kind":"declarations"}],"kind":"symbol","identifier":{"interfaceLanguage":"occ","url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonMalformed"},"schemaVersion":{"minor":3,"major":0,"patch":0},"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason"]]},"sections":[],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason/SMIChallengeReasonMalformed":{"url":"\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonmalformed","fragments":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonMalformed","title":"SMIChallengeReasonMalformed","abstract":[{"type":"text","text":"The passed in token was malformed."}],"role":"symbol","type":"topic"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"kind":"symbol","url":"\/documentation\/smiclientcore","abstract":[],"role":"collection","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","type":"topic","title":"SMIClientCore"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason":{"fragments":[{"kind":"identifier","text":"SMIChallengeReason"}],"role":"symbol","kind":"symbol","type":"topic","url":"\/documentation\/smiclientcore\/smichallengereason","title":"SMIChallengeReason","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason","abstract":[{"type":"text","text":"The reason for a credential challenge. Possible values:"}]}}}